lib/inspec/resources/ssh_config.rb

```diff
@@ -121,4 +119,99 @@ def ssh_config_file(type)
       "/etc/ssh/#{type}"
     end
   end
+
+  class SshdActiveConfig < SshdConfig
+    name "sshd_active_config"
+    supports platform: "unix"
+    supports platform: "windows"
+    desc "Use the sshd_active_config InSpec audit resource to test configuration data for the Open SSH daemon located at /etc/ssh/sshd_config on Linux and UNIX platforms. sshd---the Open SSH daemon---listens on dedicated ports, starts a daemon for each incoming connection, and then handles encryption, authentication, key exchanges, command execution, and data exchanges."
+    example <<~EXAMPLE
+      describe sshd_active_config do
+        its('Protocol') { should eq '2' }
+      end
+    EXAMPLE
+
+    attr_reader :active_path
+
+    def initialize
+      @active_path = dynamic_sshd_config_path
```

Comment:
perahaps @derived_path?

Reply:
I ended up keeping this instance variable as active_path because it coincides with the naming of the resource (sshd_ACTIVE_path), but I renamed the variable scoped within the path retrieval function to dynamic_path because it coincides with the name of the function it's in (dynamic_sshd_config_path). This way, there's not too many words being thrown around to refer to the same thing (active, dynamic, derived, etc.)